### PR TITLE
Refactor pages

### LINKED ISSUE #84: Refactor page components into a pages folder and implement user-friendly NotFoundPage
The goal of this ticket is to move page components, such as Home, Scripts, Execution, and DAGre, to a new folder src/pages, as they represent pages. It is a good React practice to separate pages from reusable components. Also implement user-friendly not found page instead of NoMatch.jsx.

### PR DESCRIPTION
**Pull request by palagdan**
Resolve #84

### CODE REVIEW

File: src/pages/Dagre/DagrePage.jsx
Comment (blcham) [inline src/pages/Dagre/DagrePage.jsx:1]: I believe this breaks convention established in the tool that directories are lower-case ... e.g. component/rest or component/sform

Comment (palagdan) [reply src/pages/Dagre/DagrePage.jsx:1]: You're right, I overlooked it.

**Comment by palagdan** [comment]
@palagdan 

To me, ScriptPage.jsx is better because it is more intuitive than Dagre. In future issues, we should refactor this component into separate ones, as the current 730-line component is very hard to understand...

**Review by blcham — APPROVED** [review]
LGTM


You are probably right but not sure where you got the convention Vocabulary.js --> vocabulary.js ...

**Pull request merged**